### PR TITLE
simple pecan app for marketplace with readme instructions for setup

### PR DESCRIPTION
**Pull request by fvukelic**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Sahil Tikale <sahil.tikale@gmail.com>
```
Comment (larsks) [inline AUTHORS]: I would vote for getting rid of the `AUTHORS` file. This is what the git commit log is for. It's too easy for an `AUTHORS` file to fall out of sync as other people work on the project.

File: ChangeLog [hunk 2]
```diff
@@ -0,0 +1,4 @@
+CHANGES
```
Comment (larsks) [inline ChangeLog]: I would drop the `ChangeLog` for the same reason. While there are valid arguments for a manually curated ChangeLog, in general we should be writing commit messages such that `git log` by itself generally provides the same information.

File: flocx_market/templates/index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<%inherit file="layout.html" />
```
Comment (larsks) [inline flocx_market/templates/index.html]: We probably don't need this: this service implement a REST api, not a web UI.

File: flocx_market/templates/layout.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<html>
```
Comment (larsks) [inline flocx_market/templates/layout.html]: Same here: unnecessary for this project.

File: flocx_market/templates/error.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<%inherit file="layout.html" />
```
Comment (fvukelic) [inline flocx_market/templates/error.html]: @larsks same for this one then I believe. Should I just remove the entire templates directory?

Comment (larsks) [reply flocx_market/templates/error.html]: Probably, yeah.  Good catch.

File: flocx_market/templates/index.html (hunk 3 above)
Comment (fvukelic) [reply flocx_market/templates/index.html]: removed and made a new PR

File: flocx_market/templates/layout.html (hunk 4 above)
Comment (fvukelic) [reply flocx_market/templates/layout.html]: removed and made a new PR

File: ChangeLog (hunk 2 above)
Comment (fvukelic) [reply ChangeLog]: done

File: AUTHORS (hunk 1 above)
Comment (fvukelic) [reply AUTHORS]: done

**Comment by larsks** [comment]
@fvukelic I made a minor edit to a commit message, but otherwise this looks fine.

**Comment by fvukelic** [comment]
great thanks!

**Pull request merged**